lib/std/start.zig

```diff
@@ -635,8 +635,10 @@ pub inline fn callMain() u8 {
                     else => {},
                 }
                 std.log.err("{s}", .{@errorName(err)});
-                if (@errorReturnTrace()) |trace| {
-                    std.debug.dumpStackTrace(trace.*);
+                if (native_os != .freestanding) {
```

Comment:
Almost any check for `freestanding` should also include `other`.

Reply:
Oh, good point; will correct.